src/tmpfiles/tmpfiles.c

```diff
@@ -601,6 +602,11 @@ static int dir_cleanup(
                                         continue;
                                 }
 
+                                if (flock(dirfd(sub_dir), LOCK_EX|LOCK_NB) < 0) {
+                                        log_debug_errno(errno, "Couldn't acquire shared BSD lock on directory \"%s\", skipping: %m", p);
```

Comment:
I think it'd be better to phrase is less like a failure: "Directory "%s" is locked, skipping."